stdlib/public/Concurrency/Task.swift

```diff
@@ -70,58 +86,58 @@ extension Task {
   }
 
-  /// Task priority may inform decisions an `Executor` makes about how and when
-  /// to schedule tasks submitted to it.
+  /// The priority of a task.
   ///
-  /// ### Priority scheduling
-  /// An executor MAY utilize priority information to attempt running higher
-  /// priority tasks first, and then continuing to serve lower priority tasks.
-  ///
-  /// The exact semantics of how priority is treated are left up to each
+  /// The executor determines how priority information impacts the way tasks are scheduled.
+  /// The behavior varies depending on the executor currently being used.
+  /// Typically, executors attempt to run tasks with a higher priority
+  /// before tasks with a lower priority.
+  /// However, the exact semantics of how priority is treated are left up to each
   /// platform and `Executor` implementation.
   ///
-  /// ### Priority inheritance
   /// Child tasks automatically inherit their parent task's priority.
   ///
-  /// Detached tasks (created by `detach`) DO NOT inherit task priority,
-  /// as they are "detached" from their parent tasks after all.
+  /// Detached tasks created by `detach(priority:operation:)` don't inherit task priority
+  /// because they aren't attached to the current task.
+  ///
+  /// In some situations the priority of a task is elevated ---
+  /// that is, the task is treated as it if had a higher priority,
+  /// without actually changing the priority of the task:
   ///
-  /// ### Priority elevation
-  /// In some situations the priority of a task must be elevated (or "escalated", "raised"):
+  /// - If a task running on behalf of an actor,
+  ///   and a new higher-priority task is enqueued to the actor,
+  ///   then the actor's current task is temporarily elevated
+  ///   to the priority of the enqueued task.
+  ///   This priority elevation allows the new task
+  ///   to be processed at (effectively) the priority it was enqueued with.
+  /// - If a task is created with a `Task.Handle`
+  ///   and a higher-priority task calls the `await handle.get()` method,
+  ///   then the priority of this task is increased until the task completes.
   ///
-  /// - if a `Task` running on behalf of an actor, and a new higher-priority
-  ///   task is enqueued to the actor, its current task must be temporarily
-  ///   elevated to the priority of the enqueued task, in order to allow the new
-  ///   task to be processed at--effectively-- the priority it was enqueued with.
-  ///   - this DOES NOT affect `Task.currentPriority()`.
-  /// - if a task is created with a `Task.Handle`, and a higher-priority task
-  ///   calls the `await handle.get()` function the priority of this task must be
-  ///   permanently increased until the task completes.
-  ///   - this DOES affect `Task.currentPriority()`.
+  /// In both cases, priority elevation prevents a low-priority task
+  /// blocking the execution of a high priority task,
+  /// also known as *priority inversion*.
```

Comment:
Hmm... I'm not sure about "prevents" it definitely "attempts to lessen the impact of" but we don't really fully prevent it... we still need to complete running the low priority task, even though now it became boosted to high priority... I guess maybe that counts as "prevents"? a bit iffy on the wording hm. @rjmccall maybe has a stronger opinion?